Game.java

```diff
@@ -1,57 +1,77 @@
 import java.util.*;
+import java.io.*;
 
 public class Game{
 
-	List<bigPlayer> lobby = new ArrayList<bigPlayer>();
-	List<bigPlayer> team1 = new ArrayList<bigPlayer>();
-	List<bigPlayer> team2 = new ArrayList<bigPlayer>();
+	List<Player> lobby = new ArrayList<Player>();
+	List<Player> team1 = new ArrayList<Player>();
+	List<Player> team2 = new ArrayList<Player>();
+	List<Map> mapList = new ArrayList<Map>();
+	List<Player> team1MapPickers = new ArrayList<Player>();
+	List<Player> team2MapPickers = new ArrayList<Player>();
+	static Scanner in = new Scanner(System.in);
+	static int MAX_LOBBY_SIZE = 10;
+	static int MAX_TEAM_SIZE = 5;
 
-	public static void main(String[] args) throws notCaptainException{
+	public static void main(String[] args) throws FileNotFoundException, notCaptainException{
```

Comment:
Is this actuall being thrown? why dont you just catch it and log/print it instead of letting it crash the program?

Reply:
Which exception are you referring to?

Reply:
Yes